vnext/.npmignore

```diff
@@ -7,7 +7,7 @@
 api-extractor.json
 build
 CMakeLists.txt
-Desktop
+/Desktop
```

Comment:
Just having Desktop seems like it prevented the JSI/Desktop folder from being included in the npm package. This stil excludes the root Desktop folder but allows JSI/Desktop to be included.